projects/plugins/jetpack/modules/custom-post-types/testimonial.php

```diff
@@ -84,7 +84,9 @@ public function maybe_register_cpt() {
 		add_filter( 'enter_title_here', array( $this, 'change_default_title' ) );
 		add_filter( sprintf( 'manage_%s_posts_columns', self::CUSTOM_POST_TYPE ), array( $this, 'edit_title_column_label' ) );
 		add_filter( 'post_updated_messages', array( $this, 'updated_messages' ) );
-		add_action( 'customize_register', array( $this, 'customize_register' ) );
+		if ( ! function_exists( 'wp_is_block_theme' ) || ( function_exists( 'wp_is_block_theme' ) && ! wp_is_block_theme() ) ) {
```

Comment:
Since `wp_is_block_theme` was introduced in 5.9, I think we can do without the check if the function exists.

Reply:
Not sure I got you.You mean we can drop the `function_exists` check?

Reply:
Sorry, I should have been clearer. I meant dropping the `function_exists` check.

Reply:
Cool. I'll revert the last commit